### PR TITLE
replace github.com/cncf/udpa/go by github.com/cncf/xds/go

### PR DESCRIPTION
**Pull request by mmorel-35**
replace `github.com/cncf/udpa/go` by `github.com/cncf/xds/go`

RELEASE NOTES:  none

Signed-off-by: Matthieu MOREL <matthieu.morel35@gmail.com>

### CODE REVIEW

**Comment by arvindbr8** [comment]
IINM,`github.com/cncf/udpa/go` is in there for backwards compatibility for management servers that may send responses with deprecated types. @dfawley -- could you please confirm?

**Comment by dfawley** [comment]
It looks like the two are equivalent now?

https://github.com/cncf/udpa/blob/c52dc94e7fbe/go/udpa/type/v1/typed_struct.go#L8

File: xds/internal/xdsclient/xdsresource/unmarshal_lds.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ import (
 	"fmt"
 	"strconv"
 
-	v1udpaudpatypepb "github.com/cncf/udpa/go/udpa/type/v1"
+	v1udpaudpatypepb "github.com/cncf/xds/go/udpa/type/v1"
```
Comment (dfawley) [inline xds/internal/xdsclient/xdsresource/unmarshal_lds.go:25]: Nit: could you change the rename to `v1xdsudpatypepb` (here and below), please?

**Review by dfawley — APPROVED** [review]

**Review by arvindbr8 — APPROVED** [review]

**Comment by arvindbr8** [comment]
@mmorel-35 -- thanks for the PR.

**Pull request merged**